src/cli/commands/install.js

```diff
@@ -257,7 +257,9 @@ export class Install {
       };
 
       pushDeps('dependencies', {hint: null, optional: false}, true);
-      pushDeps('devDependencies', {hint: 'dev', optional: false}, !this.config.production);
+      if (!this.config.production) {
```

Comment:
This breaks 11 other tests... However, I question the validity of at least one of the breaking specs:

```
 FAIL  __tests__/commands/import.js
  ● import missing dev deps in production
```